src/test/java/com/znsio/sample/e2e/screen/web/AmazonCartWeb/SearchResultsScreenWeb.java

```diff
@@ -1,2 +1,56 @@
-package com.znsio.sample.e2e.screen.web.AmazonCartWeb;public class SearchResultsScreenWeb {
+package com.znsio.sample.e2e.screen.web.AmazonCartWeb;
+
+import com.context.TestExecutionContext;
+import com.znsio.e2e.runner.Runner;
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.screen.AmazonCart.ProductDetailsScreen;
+import com.znsio.sample.e2e.screen.AmazonCart.SearchResultsScreen;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import static com.znsio.e2e.tools.Wait.waitFor;
+
+public class SearchResultsScreenWeb extends SearchResultsScreen{
+
+    private static final String NOT_YET_IMPLEMENTED = "NOT_YET_IMPLEMENTED";
+    private static final Logger LOGGER = Logger.getLogger(SearchResultsScreenWeb.class.getName());
+    private final Driver driver;
+    private final Visual visually;
+    private final TestExecutionContext context;
+    private final String SCREEN_NAME = SearchResultsScreenWeb.class.getSimpleName();
+    private final By verifySearchResult = By.xpath("//div[@class='a-section a-spacing-small a-spacing-top-small']/child::span[3]");
```

Comment:
These xpaths are bad. We should never use indices (ex: span[3]) in the locators. Find better locators